source/Halibut.Tests/Support/TestCases/ClientAndServiceTestCasesBuilder.cs

```diff
@@ -29,6 +29,13 @@ class ClientAndServiceTestCasesBuilder
 
         public IEnumerable<ClientAndServiceTestCase> Build()
         {
+            return BuildDistinct();
```

Comment:
Alternative: We could have left the `yield return` in the method body of `BuildDistince` (making it return an `IEnumerable`), then done the `Distinct` here.

All that gets us is not having to make a list, and it looks a little sexier.

So whatevs. Just thought I'd share 😁